swagger/src/test/scala/org/http4s/rho/swagger/SwaggerSupportSpec.scala

```diff
@@ -239,7 +239,7 @@ class SwaggerSupportSpec extends Specification {
     "Check metadata in API listing" in {
       val service = metaDataRoutes.toRoutes(createRhoMiddleware(swaggerRoutesInSwagger = true))
 
-      val r = Request[IO](GET, Uri(path = "/swagger.json"))
+      val r = Request[IO](GET, Uri(path = Uri.Path.fromString("/swagger.json")))
```

Comment:
I think these are going to need to be `unsafeFromString` in M20.  You can also consider the Uri interpolator: `uri"/swagger.json"`.